src/Analyzer.PowerShellRuleEngine/PowerShellRuleEngine.cs

```diff
@@ -79,7 +79,7 @@ public IEnumerable<IEvaluation> AnalyzeTemplate(TemplateContext templateContext,
             {
                 // There was an error loading the TTK module.  Return an empty collection.
 
-                logger?.LogError("There was an error running the PowerShell based checks");
+                logger?.LogWarning("There was an error running the PowerShell based checks");
```

Comment:
Do we want to change the text to not include the word error any more since it's now of type warning?

Reply:
I'm inclined to leave the messages as they are, because there was an error running PowerShell, is just that we might not consider that a total failure of the tool